### DISCUSSION TITLE
Compare SeLo and retrieval

### DISCUSSION THREAD

**Discussion by mails9523** [xiaoyuan1996/SemanticLocalizationMetrics#1] [Q&A] [answered]
Thanks for your project, I would like to ask what exactly is the difference between traditional cross-modal retrieval and selo

**Comment by xiaoyuan1996** [comment, marked as answer]
In our opinion, SeLo enables detection and segmentation at the semantic level. With sufficient data support, SeLo has the potential to unify downstream tasks. SeLo is based on cross-modal retrieval, which is a unique presentation of cross-modal retrieval in real big scene data.
